cerberus/kubernetes/client.py

```diff
@@ -125,3 +125,27 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods, failed_containers
+
+
+# Monitor cluster operators
+def monitor_cluster_operator(iteration, cluster_operators):
```

Comment:
Maybe remove the iteration parameter since we are not using it in the function?

Reply:
Fixed this in newest commit